force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -2060,7 +2060,7 @@
         <language>en_US</language>
         <protected>true</protected>
         <shortDescription>Recommended Fix when Name isn&apos;t unique in reciprocal settings</shortDescription>
```

Comment:
The short description here isnt matching the spec.

Reply:
Fixed it, thanks for catching this.